### PR TITLE
PP-615 comment can't be set on queue

### PR DESCRIPTION
**Pull request by vchlum**
#### Issue-ID

* **[PP-615](https://pbspro.atlassian.net/browse/PP-615)**

#### Problem description
* Comment can't be set on queue. It is very useful to be able to add the comment to the queues, especially when large number of the queues is in the system.

#### Cause / Analysis
* queue is missing the ATTR_comment attribute

#### Solution description
* add ATTR_comment into queue attribute definition

#### Checklist:

- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [x] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [x] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [x] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by arungrover** [comment]
@vchlum Thanks for your contribution. Since this is a new attribute that would now appear as part of queue attributes, can you please post your design proposal for this change at - [project documentation area](https://pbspro.atlassian.net/wiki/display/PD) and start a discussion on developer [forum](http://community.pbspro.org/)

Thanks!

File: src/lib/Libattr/master_queue_attr_def.xml [hunk 1]
```diff
@@ -404,6 +404,23 @@
...
+    <attributes>
+	/* QA_ATR_Comment */
+	<member_name><both>ATTR_comment </both></member_name>			<!-- "comment" --> 
+	<member_at_decode>decode_str </member_at_decode>
+	<member_at_encode>encode_str </member_at_encode>
+	<member_at_set>set_str </member_at_set>
+	<member_at_comp>comp_str </member_at_comp>
+	<member_at_free>free_str </member_at_free>
+	<member_at_action>NULL_FUNC </member_at_action>
+	<member_at_flags><both>NO_USER_SET | ATR_DFLAG_NOSAVM </both></member_at_flags>
```
Comment (mike0042) [inline src/lib/Libattr/master_queue_attr_def.xml]: This should be MGR_ONLY_SET. Is there a reason you don't want this attribute saved?

Comment (mike0042) [inline src/lib/Libattr/master_queue_attr_def.xml]: This should be MGR_ONLY_SET. Is there a reason you don't want this attribute saved?

Comment (bhroam) [inline src/lib/Libattr/master_queue_attr_def.xml]: I disagree.  The manager role is meant to be for whole decides policy.  The operator role is more for a person doing day to day maintenance.  This strikes me as something the operator should be able to set.  In any case, the node comment is NO_USER_SET.  We should be consistent.

Comment (vchlum) [inline src/lib/Libattr/master_queue_attr_def.xml]: The consistency with the node comment is the reason why I use "NO_USER_SET | ATR_DFLAG_NOSAVM". I think it is the same case as the node comment. ATR_DFLAG_NOSAVM is not necessary but I suggest to keep it consistent. Now the admin, operator and manager are able to set the queue comment, which I think is correct.

Comment (mike0042) [inline src/lib/Libattr/master_queue_attr_def.xml]: Works for me. Changes approved.

**Comment by arungrover** [comment]
Change looks good to me too

**Comment by arungrover** [comment]
All looks in place. Merging it in!

**Pull request merged**